salesagility/SuiteCRM
# Improve footer styling for new stats item.

## Description

I assume this broke when we added the stats to the footer a few days ago in #8027. This PR fixes it so everything is on one line, and removes some floats and other weird/pointless styles.

## Motivation and Context
Before:

![Screenshot_2019-10-14 SuiteCRM(1)](https://user-images.githubusercontent.com/2977353/66778366-ac963980-ee88-11e9-8b5c-8a00e5b3a0fd.png)

After:

![Screenshot_2019-10-14 SuiteCRM](https://user-images.githubusercontent.com/2977353/66778381-b5870b00-ee88-11e9-92b7-9515cf20a64c.png)

## How To Test This
1. Run a Quick Repair and Rebuild, and hard refresh the page to get the new CSS.
2. Go to any page
3. Look at the footer and see that the stats now have some padding to keep it away from the edge of the page, and the "BACK TO TOP" button is on the same line as the copyright text.

You can test multiple pages and other screen sizes as well.

## Types of changes

### Final checklist
